test/unit/spec/function_output/kbaseSampleSet-spec.js

```diff
@@ -0,0 +1,14 @@
+/*global define*/
+/*global describe, it, expect*/
+/*global jasmine*/
+/*global beforeEach, afterEach*/
+/*jslint white: true*/
+define([
+    'kbaseSampleSetView'
+], function(Widget) {
+    describe('Test the kbaseSampleSet viewer widget', function() {
+        it('Should do things', function() {
+
```

Comment:
This would be nice to have some actual tests in. I can point you to some examples of how to build unit tests for widgets.

Reply:
that would be great, Is there an example that you think is a particularly good model to use?